rom/dev/tests/rom_integration_tests/helpers.rs

```diff
@@ -116,12 +116,12 @@ pub fn change_dword_endianess(data: &mut [u8]) {
     }
 }
 
-pub fn rom_fw_id(subsystem: bool) -> &'static FwId<'static> {
-    if subsystem {
-        firmware::ss_rom_from_env()
-    } else {
-        firmware::rom_from_env()
-    }
+pub fn model_supports_subsystem_mode(subsystem_mode: bool) -> bool {
```

Comment:
nit: I'm worried the name of this function could be misread. When I first read it I read "Does the model support running in subsystem mode?" so `((!subsystem_mode) && fpga_realtime)` seemed incorrect. Maybe a better name would be `model_supports_subsystem_config`,  `*_setting`, or something similar

Reply:
Cool. I can rename it. Getting the boolean logic to be readable and naming is the hardest part. :)